### PR TITLE
Added Information About Deleting Subscribers From Grid

### PR DESCRIPTION
**Pull request by DmitryTsymbal**
## Purpose of this pull request

Added Information About Deleting Subscribers From Grid

This pull request (PR) ...

## Magento release version

_Which Magento release(s) are affected by the content changes: 2.4, 2.3? Specify a patch release number, if applicable._

## Product editions

_Is this update specific to a product edition: Magento Open Source only, Adobe Commerce only?_

- ...

_Is this update specific to an installed feature extension: B2B extension, other feature set (please, specify)?_

- ...

## Affected documentation pages

https://docs.magento.com/user-guide/marketing/newsletter-subscribers.html

- ...
- ...

### CODE REVIEW

File: src/marketing/newsletter-subscribers.md [hunk 1]
```diff
@@ -70,3 +70,13 @@ If you want to use a third-party service to send newsletters, you can export you
 
    ![Export newsletter subscribers]({% link images/images/newsletter-subscribers-export.png %}){: .zoom}
    _Export Newsletter Subscribers_
+
+## Deleting a Subscriber from Newsletter Subscribers list
```
Comment (leslietilling) [inline src/marketing/newsletter-subscribers.md:74]:
```suggestion
## Delete a subscriber from the Newsletter Subscribers list
```

File: src/marketing/newsletter-subscribers.md [hunk 2]
```diff
@@ -70,3 +70,13 @@ If you want to use a third-party service to send newsletters, you can export you
...
+
+1. Navigate to **Marketing** > _Communications_ > **Newsletter Subscribers**.
```
Comment (leslietilling) [inline src/marketing/newsletter-subscribers.md:76]:
```suggestion
1. On the _Admin_ sidebar, go to **Marketing** > _Communications_ > **Newsletter Subscribers**.
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Just some minor edits for style and consistency

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the pull request successfully imported.

**Pull request merged**